editor/editor_settings.cpp

```diff
@@ -551,6 +551,12 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	EDITOR_SETTING(Variant::INT, PROPERTY_HINT_RANGE, "text_editor/behavior/indent/size", 4, "1,64,1") // size of 0 crashes.
 	_initial_set("text_editor/behavior/indent/auto_indent", true);
 
+	// Behavior: Formatter
+	_initial_set("text_editor/behavior/formatter/maximum_line_length", 80);
```

Comment:
We should change the default to 100. It's sensible to suggest users to keep it 80 or less, but this doesn't make sense to enforce automatically. Game logic in GDScript will naturally have some long lines.

The script editor already shows a faint line at 80 characters and a thicker line at 100 characters. I think the character count used by the formatter should be the same as this thicker line. Actually, what we should likely do is unify these settings. `text_editor/appearance/guidelines/line_length_guideline_hard_column`